Understanding kata-fc (#45454)
kubernetes/website: content/en/docs/concepts/scheduling-eviction/pod-overhead.md

You need to make sure a `RuntimeClass` is utilized which defines the `overhead` field.

> ## Understanding "kata-fc" 

Reviewer: This shouldn't have it's own section. Can you integrate the explanation into the first paragraph of the example section instead? It shouldn't be more than a sentence either, as it's not really relevant to the example or the rest of the article.

Author: @tallclair Yeah sure, I will make those changes.

To work with Pod overhead, you need a RuntimeClass that defines the `overhead` field. As
an example, you could use the following RuntimeClass definition with a virtualization container

> runtime like 'kata-fc'. The "kata" prefix from the 'Kata Containers' project, designed for lightweight container virtualization, with the "fc" suffix 
> representing the integration of the 'Firecracker' VMM (Virtual Machine Monitor) that uses around 120MiB per Pod for the virtual machine and the guest OS:

Reviewer 2: Try this:
```suggestion
runtime (in this example, Kata Containers) that uses around 120MiB per Pod for the virtual machine
and the guest OS:
```

```yaml

> apiVersion: node.k8s.io/v1

Reviewer 2: ```suggestion
# You need to change this example to match the actual runtime name, and per-Pod
# resource overhead, that the container runtime is adding in your cluster.
apiVersion: node.k8s.io/v1
```
